src/environment/Environment.h

```diff
@@ -15,8 +15,7 @@
  *   You should have received a copy of the GNU Affero General Public License
  *   along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
-#ifndef SRC_ENVIRONMENT_ENVIRONMENT_H_
-#define SRC_ENVIRONMENT_ENVIRONMENT_H_
+#pragma once

```

Comment:
Please use normal include guards as this pragma is non-standard.

Reply:
Sorry about the include guards. We discussed this issue some months ago, and I had the understanding that you were OK to use pragma once, since all the compilers in use supported it. I will change them back to the old C-style.